tsconfig.json

```diff
@@ -10,12 +13,14 @@
     "noImplicitAny": false,
     "removeComments": true,
     "noUnusedLocals": true,
-    "lib": [
-      "esnext"
-    ],
-    "rootDir": "./src"
+    "rootDir": ".",
+    "resolveJsonModule": true,
+    "newLine": "lf",
+    "noFallthroughCasesInSwitch": true,
+    "isolatedModules": true,
+    "forceConsistentCasingInFileNames": true,
+    "skipLibCheck": true,
+    "lib": ["ESNext"]
   },
-  "include": [
-    "src"
-  ]
-}
\ No newline at end of file
+  "include": ["src", "test"]
```

Comment:
Please remove the `test` here.
It is used to build the packages, and should not include `test`.